src/OrchardCore.Modules/OrchardCore.ReCaptcha/Manifest.cs

```diff
@@ -0,0 +1,27 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "ReCaptcha",
+    Author = "The Orchard Team",
+    Website = "http://orchardproject.net",
+    Version = "2.0.0")]
+
+[assembly: Feature(
+    Id = "OrchardCore.ReCaptcha",
+    Name = "ReCaptcha",
+    Category = "Security",
+    Description = "Provides ReCaptcha functionality.")]
+
+[assembly: Feature(
```

Comment:
Indeed, we need three features, I forgot the one on the forgot password page.
When we extend the RegisterPage, the shapes of the register page have to be registered.
I do not want to activate registration just because I activated recaptcha.

tl/dr; for each user feature, we need a seperate recaptcha feature

Reply:
We now have one feature with settings.